packages/discovery-provider/ddl/migrations/0077_backfill_user_socials.sql

```diff
@@ -0,0 +1,87 @@
+begin;
+
+do $$
+begin
+    -- run only on stage
+    if exists (select * from "blocks" where "blockhash" = '0x65a3243860511ed28a933c3a113dea7df368ad53f721cc9d0034c0c75f996afb') then
+
+        -- check whether the data has already been backfilled and return early for idempotency
+        if exists (select 1 from users where website is not null) then
```

Comment:
There are a lot of empty values in the input CSV. Would those not result in `null` for the websites in a lot of these rows and cause this script to run every time?

Reply:
Any `user` row with a non-null website would cause this to retry early...
It does seem like prod currently has no user rows with a `website` value... so this seems okay.

Reply:
Oh! I read too quickly and didn't notice this was looking for any valid website field in any row. Carry on...